### ISSUE TITLE
Colab installation is being blocked by statsmodel 0.13.2

### ISSUE THREAD

**Issue by horizon-blue** [facebookresearch/beanmachine#1340]
### Issue Description

TL;DR: Bean Machine cannot be installed anymore because pip is getting stuck at installing statsmodels 0.13.2 forever:

![image](https://user-images.githubusercontent.com/18493382/153301997-1509028d-28eb-4f9b-8782-88a99c544c1c.png)

### Steps to Reproduce
To reproduce, simply run any of the tutorials on our website: https://beanmachine.org/docs/tutorials/

Alternatively, create an empty Colab notebook, and run `!pip install beanmachine`. Or even simpler, run `!pip install -U statsmodels` (which was the root cause of the installation issue).

### System Info
- Environment: Google Colab
- PyTorch Version: 1.10.0
- Python version: 3.7.12 (on google Colab)

### Additional Context

Related issue on statsmodels repo: https://github.com/statsmodels/statsmodels/issues/8110

Statsmodel just made their 0.13.2 release yesterday, which does not include a Python 3.7 wheel. However, the PyPI package index still mark the package as Python>=3.7. One workaround we can do is to pin the statsmodels at 0.13.1 for now

**Comment by ndmlny-qs** [comment]
Just ran the `Linear_Regression` tutorial and can confirm the error and workaround is successful. Would updating the documentation to indicate the need to pip install the pinned version of statsmodels require a full release?

**Comment by horizon-blue** [comment]
Seems like statsmodels folks have resolved the issue on their side by publishing Python 3.7 wheels, so we no longer need to do anything from our side :).
